buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijPlugin.kt

```diff
@@ -35,18 +36,59 @@ fun Project.configureIntellijPlugin() {
     tasks.withType<BuildSearchableOptionsTask> { enabled = false }
   }
 
+  // This task should not be used - we don't use the "Unreleased" section anymore
+  project.gradle.startParameter.excludedTaskNames.add("patchChangeLog")
+
   configure<ChangelogPluginExtension> {
     val PROSPECTIVE_RELEASE_VERSION: String by extra
     version.set("v$PROSPECTIVE_RELEASE_VERSION")
-    header.set(version)
-    headerParserRegex.set(Regex("v\\d+\\.\\d+\\.\\d+"))
+    headerParserRegex.set(Regex("""v\d+\.\d+\.\d+"""))
     path.set("${project.projectDir}/CHANGE-NOTES.md")
-    unreleasedTerm.set("Unreleased")
-    groups.set(emptyList())
   }
 
   val changelog = extensions.getByType(ChangelogPluginExtension::class.java)
 
+  val verifyStructureTask = tasks.register("verifyChangeLogStructure") {
+    doLast {
+      val prospectiveVersionSection = changelog.get(changelog.version.get())
+      val latestVersionSection = changelog.getLatest()
+
+      if (prospectiveVersionSection.version != latestVersionSection.version) {
```

Comment:
Whoops I changed latest the version in CHANGE-NOTES.md from v3.2.0 to v3.2.1 and run `./gradlew verifyChangeLog`... got:

```

FAILURE: Build failed with an exception.

* Where:
Build file '/home/plipski/git-machete-intellij-plugin/build.gradle.kts' line: 228

* What went wrong:
Version is missing: [Unreleased]
> Version is missing: [Unreleased]

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2s
15 actionable tasks: 15 up-to-date
```

which is an unexpected error 🤔